src/osd/SnapMapper.cc

```diff
@@ -881,6 +887,11 @@ bool SnapMapper::is_legacy_mapping(const string &to_test)
     LEGACY_MAPPING_PREFIX;
 }
 
+bool SnapMapper::is_malformed_mapping(const string &to_test)
```

Comment:
(string_view to_test)

Reply:
This function calls is_mapping() which gets const string&, I don't want to change any existing code to avoid conflicts if this will be backported.